modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -756,6 +756,28 @@ static PyObject* pyopencv_cv_gapi_kernels(PyObject* , PyObject* py_args, PyObjec
     return pyopencv_from(pkg);
 }
 
+static PyObject* pyopencv_cv_gapi_networks(PyObject*, PyObject* py_args, PyObject*)
+{
+    using namespace cv;
+    gapi::GNetPackage pkg;
+    Py_ssize_t size = PyTuple_Size(py_args);
+    for (int i = 0; i < size; ++i)
+    {
+        PyObject* item = PyTuple_GetItem(py_args, i);
+        if (PyObject_TypeCheck(item,
```

Comment:
Consider using `pyopencv_to_safe`

Reply:
Or `pyopencv_to` in that you won't throw exception manually

Reply:
Added `pyopencv_to`. Is it correct change?
Error message with wrong type is:
`TypeError: Expected cv::gapi::ie::PyParams for argument 'Params'`